allow placeholders in extensibility (#1783)
kyma-project/busola: docs/extensibility/form-widgets.md

<img src="./assets/form-widgets/Text.png" alt="Example of a text widget" style="border: 1px solid #D2D5D9">

#### Widget-specific parameters

> - **placeholder** - specifies a short hint that describes the expected value of an input field.

Reviewer: ```suggestion
- **placeholder** - specifies a short hint about the input field value.
```
The same applies to remaining placeholder definitions.
